docker-compose-dependencies.yml

```diff
@@ -2,8 +2,18 @@
 version: '2.1'
 
 services:
+
+  rpa-native-pdf-annotator-db:
```

Comment:
I agree with moving the app's db in here. it's a dependency, doesn't belong in docker-compose.yml

Reply:
it's required when you run the dependencies, otherwise you'd have to spin up docker twice.